apps/wallet/src/background/accounts/index.ts

```diff
@@ -259,5 +259,16 @@ export async function accountsHandleUIMessage(msg: Message, uiConnection: UiConn
 		await uiConnection.send(createMessage({ type: 'done' }, msg.id));
 		return true;
 	}
+	if (isMethodPayload(payload, 'verifyPassword')) {
+		const allAccounts = await getAllAccounts();
+		for (const anAccount of allAccounts) {
+			if (isPasswordUnLockable(anAccount)) {
+				await anAccount.verifyPassword(payload.args.password);
+				await uiConnection.send(createMessage({ type: 'done' }, msg.id));
+				return true;
```

Comment:
this will always return true - was that intended?

Reply:
yes that's intended, returning true means the message was handled (probably should find a better way to do this)